packages/simplebar-core/src/index.ts

```diff
@@ -370,7 +371,9 @@ export default class SimpleBarCore {
     this.el.addEventListener('mousemove', this.onMouseMove);
     this.el.addEventListener('mouseleave', this.onMouseLeave);
 
-    this.contentWrapperEl?.addEventListener('scroll', this.onScroll);
+    this.contentWrapperEl?.addEventListener('scroll', this.onScroll, {
+      passive: true
```

Comment:
Thanks for your contribution! Could you just remove this? (and maybe create a new PR for this specifically?)

Reply:
I just updated the PR :)